### PR TITLE
Display by week

### PR DESCRIPTION
**Pull request by lizziesz**

### CODE REVIEW

File: test/e2e/features/step-definitions/bookings-list.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { defineSupportCode } from 'cucumber'
```
Comment (lizziesz) [inline test/e2e/features/step-definitions/bookings-list.js]: This cucumber test is adding a booking for next week, and checking the week spinner by navigating to next week and seeing if that booking is displayed. There could be a concern that until cancelling a booking gets hooked up to the tests, this test could crowd the database with bookings, so I am open to suggestions/commenting this test out until cancelling gets integrated.

Comment (aochsner) [reply test/e2e/features/step-definitions/bookings-list.js]: for now, let's leave it and see what happens...

File: test/e2e/features/step-definitions/bookings-list.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    await endInput.clear()
+    await endInput.sendKeys(endForForm)
+    const createButton = await driver.findElement(By.tagName('button'))
+    await createButton.click()
+  })
+
+  When('I view my bookings and navigate to next week', async () => {
+    await driver.get(url)
+    await driver.findElement(By.linkText('View Your Bookings')).click()
+    await driver.sleep(1000)
```
Comment (aochsner) [inline test/e2e/features/step-definitions/bookings-list.js]: any chance we can avoid this and change it to a `wait`?

File: test/e2e/features/step-definitions/bookings-list.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    const startInput = await driver.findElement(By.name('start'))
+    await driver.wait(async () => (await startInput.getAttribute('value')).includes('T'))
+    await startInput.clear()
+    await startInput.sendKeys(startForForm)
+    const endInput = await driver.findElement(By.name('end'))
+    await driver.wait(async () => (await endInput.getAttribute('value')).includes('T'))
+    await endInput.clear()
+    await endInput.sendKeys(endForForm)
+    const createButton = await driver.findElement(By.tagName('button'))
+    await createButton.click()
```
Comment (aochsner) [inline test/e2e/features/step-definitions/bookings-list.js]: might want to refactor this as there's a lot of duplication with `Given I create a booking`

**Pull request merged**